randovania/games/fusion/exporter/options.py

```diff
@@ -10,10 +10,9 @@
 
 @dataclasses.dataclass(frozen=True)
 class FusionPerGameOptions(PerGameOptions):
-    """ """
-
     input_path: Path | None = None
     output_path: Path | None = None
+    output_format: str = "gba"
```

Comment:
a todo option to automatically create a patch when exporting

Reply:
do we even want to support people using patch files instead of RDV games?

Reply:
it's a common enough operation that it's probably worth having.